exercises/bowling/example.tt

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'bowling'
+
+# Test data version:
+# <%= sha1 %>
+class BowlingTest < Minitest::Test
+  def setup
+    @game = Game.new
+  end
+
+  def roll(rolls)
+    rolls.each { |pins| @game.roll(pins) }
+  end
+<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
```

Comment:
Try to keep logic out of the template file.
Have a look at the PR for anagram generator: https://github.com/exercism/xruby/pull/464 for some examples of how to do this.

Basically the test template should look like:

```ruby
  def <%= test_case.test_name %>
    <%= test_case.skipped %>
    <%= test_case.work_load %>
  end
```